### PR TITLE
fix: both axis with continuous values

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #2790 

### Description:

Changes:
- only force type 'category' for horizontal bar chart
- apply correct type on each axis top/bottom/left/right

Screenshot 2022-08-03 at 15 38 47

### CODE REVIEW

File: packages/frontend/src/hooks/echarts/useEcharts.ts [hunk 1]
```diff
@@ -68,6 +68,60 @@ const getAxisTypeFromField = (item?: Field): string => {
...
 
+type GetAxisTypeArg = {
+    validCartesianConfig: CartesianChart;
+    itemMap: Record<string, Field | TableCalculation>;
+    topAxisXId?: string;
+    bottomAxisXId?: string;
+    rightAxisYId?: string;
+    leftAxisYId?: string;
+};
+const getAxisType = ({
```
Comment (rephus) [inline packages/frontend/src/hooks/echarts/useEcharts.ts:79]: beautiful

**Review by TuringLovesDeathMetal — APPROVED** [review]
it's beautiful 😍 having it as a value on the x-axis also makes it so that you don't have to `order by` every time! 

amazing 😚 👌

File: packages/frontend/src/hooks/echarts/useEcharts.ts (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/hooks/echarts/useEcharts.ts:79]: Is this a new bug with datetimes ? 
image

Comment (ZeRego) [reply packages/frontend/src/hooks/echarts/useEcharts.ts:79]: Nice catch! We can support 'date' type on the y axis for horizontal bars. We only need to force 'category' when the default type is 'value'.

e.g of date in the y axis:
Screenshot 2022-08-03 at 16 27 18

**Pull request merged**